lib/dpl/provider/npm.rb

```diff
@@ -22,7 +22,8 @@ def setup_auth
 
       def check_auth
         setup_auth
-        log "Authenticated with email #{option(:email)}"
+        api_key = option(:api_key).gsub(/[^-]/, '*')[0..-5] + option(:api_key)[-4..-1]
```

Comment:
Something more simple such as 
```ruby
api_key = option(:api_key).gsub(/.*(....)$/, '....\1')
```

might be OK. In the above, the length of API key length is lost, but perhaps that's not important information for debugging purposes.

If it is, I think we should put this in a separate method to make it easier to read.